src/autogluon/bench/entrypoint.sh

```diff
@@ -1,3 +1,10 @@
 #!/bin/bash
 
-agbench run $config_file
+
+if [ -n "$AG_BENCH_DEV_URL" ]; then
+    echo "Using Development Branch: $AG_BENCH_DEV_URL" >&2
+    agbench run $config_file --dev-branch $AG_BENCH_DEV_URL
+else
+    echo "Using Released autogluon.bench: " >&2
+    agbench run $config_file
+fi
```

Comment:
Not a big deal, but we should add a new-line at the end of this file when possible.